### PR TITLE
Add error page and redirect to new page on poster generation error

### PR DESCRIPTION
**Pull request by ibarrae**

### CODE REVIEW

File: src/pages/error.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+)
+
+export const getServerSideProps: GetServerSideProps<ErrorPageProps> = async ({
+  req,
+}) => {
+  const session = await auth0.getSession(req)
+  return {props: {isLoggedIn: Boolean(session)}}
+}
+
+type ErrorPageProps = {isLoggedIn: boolean}
```
Comment (JoseRFelix) [inline src/pages/error.tsx:46]: Not a blocker, this type could be converted to an interface and can be placed before the component declaration. In my opinion, it improves readability and reduces cognitive load by not making the dev look into the bottom of the files for the props. What do you think?

File: api/posters/start/start.ts [hunk 2]
```diff
@@ -149,12 +149,7 @@ export function startImplementation(event: SQSEvent) {
       logger.error(e)
 
       try {
-        await PosterModel.update(
-          {posterSlug, userId},
-          {
-            step: PosterSteps.FAILED,
-          },
-        )
+        await sendUpdateToClient(posterSlug, userId, PosterSteps.FAILED)
```
Comment (JoseRFelix) [inline api/posters/start/start.ts:152]: :+1:

**Review by JoseRFelix — APPROVED** [review]
Looks good! Left one comment

File: src/pages/error.tsx (hunk 1 above)
Comment (ibarrae) [reply src/pages/error.tsx:46]: For sure, that makes a lot of sense. Will update it accordingly!

**Pull request merged**